MIGRATION.md

```diff
@@ -48,6 +49,28 @@ git pull
 pip install -e .
 ```
 
+### Install amdsmi Package (Strongly Recommended)
+
+For best results, install the `amdsmi` Python package alongside pyrsmi:
```

Comment:
```suggestion
If not using conda, install the `amdsmi` Python package alongside pyrsmi:
```
(I haven't managed to explain exactly what conda uses automatically that other pyenvs don't, so this change at least notes the difference / encourages conda.)